### ISSUE TITLE
automation failure code is still not correct

### ISSUE THREAD

**Issue by aclum** [microbiomedata/nmdc_automation#690]
We had some failures in dev over the weekend and the agent.state file says the failure count is 2 but jaws is showing 3 failed jobs.

See 

      "claims": [],
      "opid": "nmdc:sys0tmv3za91",
      "done": true,
      "start": "2026-03-14T01:20:57.906988+00:00",
      "jaws_jobid": 170968,
      "last_status": "failed",
      "nmdc_jobid": "nmdc:sys027kbwm69",
      "failed_count": 2
    }

JAWS IDs: 170968, 170962, 170944
